### PR TITLE
release(v0.11.0-beta.0): prepare release

### PR DESCRIPTION
**Pull request by smira**
This is the official v0.11.0-beta.0 release.

Signed-off-by: Andrey Smirnov <smirnov.andrey@gmail.com>

### CODE REVIEW

File: hack/release.toml [hunk 1]
```diff
@@ -75,7 +76,8 @@ Default `talosconfig` has `os:admin` role embedded in the certificate so that al
 Certificates with reduced set of roles can be created with `talosctl config new` command.
 
 When upgrading from Talos 0.10, RBAC is not enabled by default. Before enabling RBAC, generate `talosconfig` with
-`os:admin` role first to make sure that administrator still has access to the cluster when RBAC is enabled.
+`os:admin` role first to make sure that administrator still has access to the cluster when RBAC is enabled
+(usin `talosctl config new` command).
```
Comment (Unix4ever) [inline hack/release.toml:80]:
```suggestion
(using `talosctl config new` command).
```

**Review by Unix4ever — APPROVED** [review]

File: hack/release.toml (hunk 1 above)
Comment (smira) [reply hack/release.toml:80]: how come I missed that, thanks, I'll repush that

**Review by andrewrynhard — APPROVED** [review]

**Pull request merged**